src/app/[locale]/viewer/_components/submodel/SubmodelDetailComponentProps.ts

```diff
@@ -0,0 +1,6 @@
+import { Submodel } from '@aas-core-works/aas-core3.0-typescript/types';
+
```

Comment:
(comment on double newline, has nothing to do with this PR) The autoformat removes this line, it might would make sense to run prettier as pre-commit hook. Prettier is already installed in the repo 🤔

Reply:
Agreed! But this is for a new ticket :)!